src/common/helper.js

```diff
@@ -651,8 +649,8 @@ module.exports = {
     }
     return ensureUbhanUserId({ userId })
   },
-  getM2Mtoken,
-  getTopcoderM2MToken,
+  getM2MToken,
```

Comment:
Export  functions `getM2MToken` and `getM2MUbahnToken` though the functions are currently being used merely inside `helper.js`